api_v3/lib/KalturaResponseCacher.php

```diff
@@ -672,6 +683,22 @@ public static function rateLimit($service, $action, $params, $partnerId = null,
 
 		return false;
 	}
+	
+	public static function incrementPostCallRateLimit($params, $partnerId = null, $ksPartnerId = null)
+	{
+		$rule = self::getRateLimitRule($params, $partnerId, $ksPartnerId);
```

Comment:
why do we need to check the rule again? we can just check if we stored something in self::$generatedKey